demo.py

```diff
@@ -49,14 +48,8 @@ def drive(track, offset):
             assert len(status) == 794, len(status)
             absPosX, absPosY, absPosZ = struct.unpack_from('fff', status, 44)
             angX, angY, angZ = struct.unpack_from('fff', status, 56)
-            num_cars = struct.unpack_from('B', status, 128)
-            posX = struct.unpack_from('f', status, 168)[0]
-            posY = struct.unpack_from('f', status, 324)[0]
-            speed = struct.unpack_from('f', status, 480)[0]
```

Comment:
Well, they are not. See the commit in Roborace Dreams. So far I have updated only the main position of the robot and not the opponents. I have removed the variables since they were not used anywhere.